src/views/PatronPreRegistrationRecordsDuplicates/PatronPreRegistrationRecordsDuplicates.js

```diff
@@ -100,7 +100,7 @@ const PatronPreRegistrationRecordsDuplicates = ({
           style={{ height: wrapperHeight }}
         >
           <PreRegistrationRecordsDuplicatesList
-            isLoading={isLoading || isUsersLoading}
+            isLoading={isLoading || isFetching}
```

Comment:
looks strange isLoading || isFetching, could you please add more reasonable names? isUsersLoading was fine, but what another isLoading in this context is not clear